usr/share/rear/prep/GNU/Linux/240_include_multipath_tools.sh

```diff
@@ -1,7 +1,9 @@
 # 240_include_multipath_tools.sh
 # Boot Over SAN executables and other goodies
 
-[[ $BOOT_OVER_SAN != ^[yY1] ]] && return
+if ! is_true "$BOOT_OVER_SAN" ; then
```

Comment:
In general:
Caution with 'is_false' versus '! is_true'!
Both tests are not identical, see the comments
for the is_true and is_false functions in
lib/global-functions.sh and see also
https://github.com/rear/rear/pull/1212#issuecomment-283698753